uroot/root.go

```diff
@@ -149,14 +140,6 @@ func Rootfs() {
 		}
 	}
 
-	for _, d := range devs {
```

Comment:
I also cleaned up this piece of unused code. It appears `dev.cpio` covers everything `mknod` can do.